### PR TITLE
Edit cutting explanation

### PR DESCRIPTION
**Pull request by ibrahim-shehzad**
Correct the $\theta$ parameter in the explanatory [material ](https://qiskit-extensions.github.io/circuit-knitting-toolbox/circuit_cutting/explanation/index.html#equation-eq-qpd)but replacing it with $-\theta/2$.

### CODE REVIEW

**Comment by garrison** [comment]
Thanks for catching this and submitting a PR.  This error is due to a convention mismatch: there is a prefactor in the exponential in the definition of the [RZZGate](https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.library.RZZGate), but this was not in the original gate cutting paper.  The code correctly accounts for this conversion, but the documentation does not until we merge the current PR.

https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/blob/a9bfc91b9855a78fd6b0dd9ffadcd03069808fc7/circuit_knitting/cutting/qpd/decompositions.py#L393-L395

File: docs/circuit_cutting/explanation/index.rst [hunk 1]
```diff
@@ -60,10 +60,10 @@ As a basic and explicit example, let us consider the decomposition of a cut :cla
...
+1. With coefficient :math:`a_1 = \cos^2 (\theta/2)`, do nothing (:math:`I \otimes I`, where :math:`I` is the identity operation on a single qubit).
+2. With coefficient :math:`a_2 = \sin^2 (\theta/2)`, perform a :class:`~qiskit.circuit.library.ZGate` on each qubit (:math:`Z \otimes Z`).
+3. With coefficient :math:`a_3 = -\sin\theta/2`, perform a projective measurement in the Z basis on the first qubit and an :class:`~qiskit.circuit.library.SGate` gate on the second qubit (denote this as :math:`M_z \otimes S`).  If the result of the measurement is 1, flip the sign of that outcome's contribution during reconstruction.
+4. With coefficient :math:`a_4 = \sin \theta/2`, perform a projective measurement in the Z basis on the first qubit and an :class:`~qiskit.circuit.library.SdgGate` gate on the second qubit (denote this as :math:`M_z \otimes S^\dagger`).  If the result of the measurement is 1, flip the sign of that outcome's contribution during reconstruction.
```
Comment (garrison) [inline docs/circuit_cutting/explanation/index.rst:66]: I believe restoring the parentheses will remove ambiguity.

no-parentheses


```suggestion
3. With coefficient :math:`a_3 = -\sin(\theta)/2`, perform a projective measurement in the Z basis on the first qubit and an :class:`~qiskit.circuit.library.SGate` gate on the second qubit (denote this as :math:`M_z \otimes S`).  If the result of the measurement is 1, flip the sign of that outcome's contribution during reconstruction.
4. With coefficient :math:`a_4 = \sin(\theta)/2`, perform a projective measurement in the Z basis on the first qubit and an :class:`~qiskit.circuit.library.SdgGate` gate on the second qubit (denote this as :math:`M_z \otimes S^\dagger`).  If the result of the measurement is 1, flip the sign of that outcome's contribution during reconstruction.
```

Comment (ibrahim-shehzad) [reply docs/circuit_cutting/explanation/index.rst:66]: I did think that I added parenthesis in the most recent commit ([89a76bf](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/pull/657/commits/89a76bf56279e3feaefdf01dfdb71a9c2bd7bb05))

Comment (ibrahim-shehzad) [reply docs/circuit_cutting/explanation/index.rst:66]: Sorry, I actually missed these ones. My bad.

**Review by garrison — APPROVED** [review]
Looks great, thank you :100:

**Pull request merged**